sklearn/ensemble/forest.py

```diff
@@ -990,6 +989,7 @@ class labels (multi-output problem).
     --------
     DecisionTreeClassifier, ExtraTreesClassifier
     """
+
```

Comment:
I don't think this is a convention we adhere to, even though I see pep257 asks for it.

Reply:
I think the """ leaves enough space